src/layouts/EditHomepage/EditHomepage.jsx

```diff
@@ -1082,6 +1237,54 @@ const EditHomepage = ({ match }) => {
                                       />
                                     </Editable.DraggableAccordionItem>
                                   )}
+
+                                  {section.announcements &&
+                                    errors.sections[sectionIndex]
+                                      .announcements && (
```

Comment:
is this intended? this seems to only render if there's an error at that specific section

Reply:
Good catch, this was due to an error that I was facing where the `errors.sections[sectionIndex].announcements` was set to `undefined` for 2 renders. I wasnt able to pinpoint the exact reason here, but had to timebox the effort here + fix it temp but wrapping it around a check.